docs/releases/v2019.1.rst

```diff
@@ -0,0 +1,219 @@
+Gluon 2019.1
+############
+
+Important notes
+***************
+
+If you are upgrading from a version prior to v2018.1 please note that the flash layout on some 
+devices (TP-Link CPE/WBS 210/510) was changed. To avoid upgrade failures, make sure to upgrade
+to Gluon 2017.1.8 or the latest Gluon 2016.2.x (unreleased) before installing Gluon 2018.1, 2018.2 or 2019.1.
+
+Routers will now only respond to queries on ``[ff05::2]:1001``, this will most likely required
+updates to your respondd queriers. This change was required due to cross-domain leakage of respondd data.
+
+Added hardware support
+**********************
+
+ar71xx-generic
+==============
+
+* D-Link
+
+  - DAP-1330 A1
+
+* TP-Link
+
+  - CPE210 v3
+
+
+ar71xx-tiny [#deprecated]_
+==========================
+
+* TP-Link
+
+  - TL-WR840N v2
+
+ipq40xx
+=======
+
+* 8devices
+
+  - Jalapeno
+
+mpc85xx-p1020 [#newtarget]_
+===========================
+
+* Aerohive
+
+  - HiveAP 330
+
+ramips-mt7621
+=============
+
+* ASUS
+
+  - RT-AC57U [#noibss]_
+
+.. [#deprecated]
+  This target will be reaching its end of life soon.
+
+.. [#newtarget]
+  This is a new target.
+
+.. [#noibss]
+  Device or target does not support AP+IBSS mode: This device or target will not be built
+  when *GLUON_WLAN_MESH* is set to ``ibss``.
+
+
+.. note::
+
+    The ``ipq806x`` target has been flagged as broken, as all of its devices are not yet fully supported in this OpenWrt release. You might have to update your build scripts accordingly.
+
+
+
+New features
+************
+
+batman-adv coexistence
+======================
+
+To allow a migration from B.A.T.M.A.N. compat 14, this Gluon release offers the ability to ship both B.A.T.M.A.N adv compat versions 14 and 15 in the same image. The ``mesh.batman_adv.routing_algo`` option decides which module gets loaded and the scheduled domain switching functionality can be used to migrate between the two versions.
+
+See the :ref:`mesh <user-site-mesh>` section for the *site.conf* configuration of this feature.                  
+
+Outdoor Mode
+============
+
+Radio devices hosted outdoors are often affected by different regulation than if they were installed indoors. The outdoor mode allows for the reconfiguration of 5 GHz radios onto different channels and channel ranges. Honoring stipulations like DFS currently make it difficult to use the 5 GHz band for mesh networking because it's never certain that nodes would stay on the same channel.
```

Comment:
I would remove the "would" in the last sentence.

Reply:
Disagree, what do you think @bobcanthelpyou?

Reply:
Maybe: `it's never certain that nodes will stay on the same channel.` ?

Reply:
@skorpy2009 rather than downvote a solution you don't like you could propose or vote for a solution you like. That would be much appreciated.